### DISCUSSION TITLE
Best batch size for the API

### DISCUSSION THREAD

**Discussion by jonathanantoine** [trakt/trakt-api#259] [Q&A] [answered]
Hello,

My app send "set as seen"/"set as not seen" commands to the API by batch of 100 items. 
If it does not succeeds (= not a 200 status code answer) I send the whole batch again (later).

Is this size of batch (100 items) a correct value for the sizing of the API? Do you need me to reduce it ?

Thanks a lot !

ps : @rudf0rd I am looking for an explanation of these spam !

**Comment by rectifyer** [comment, marked as answer]
In general 100 should be ok, but it really depends what type of items are being sent. For example, sending 100 episodes is way less data then sending 100 shows. If you're sending episodes and movies, then yes I think 100 is an ok number.

**Reply by jonathanantoine** [reply]
Thank you!
